testing/acceptance_test.py

```diff
@@ -241,7 +241,7 @@ def test_issue88_initial_file_multinodes(self, pytester: Pytester) -> None:
         pytester.copy_example("issue88_initial_file_multinodes")
         p = pytester.makepyfile("def test_hello(): pass")
         result = pytester.runpytest(p, "--collect-only")
-        result.stdout.fnmatch_lines(["*Module*test_issue88*", "*MyFile*test_issue88*"])
```

Comment:
This existing test and the one below actually flagged the issue in #11646, but I misdiagnosed it and accepted the changes. So this brings the tests to how they were.

Reply:
Probably leave a comment making the intent of the test more explicit?

Reply:
This test failing was mostly incidental, it does not aim specifically to test ordering but something with custom nodes. I added the new test to check the order specifically which will be much harder to misdiagnose.